composer.json

```diff
@@ -22,8 +22,8 @@
         }
     ],
     "require": {
-        "php": ">=7.2|^8.0",
-        "illuminate/support": "^6.0|^7.0|^8.0|^9.0|^10.0",
+        "php": "^8.1",
+        "illuminate/support": "^10.0",
         "rollbar/rollbar": "^2.0 | ^3.1"
```

Comment:
I think we may want to drop support for `rollbar/rollbar` v2 as it is not compatible with PHP 8.

Reply:
..and you're right it's not possible to support Laravel 9.x and below, as they only support monolog v2 and not v3.

Reply:
> I think we may want to drop support for `rollbar/rollbar` v2 as it is not compatible with PHP 8.

Removed older 2x